unittests/test_environments.py

```diff
@@ -88,6 +88,15 @@ def test_load_restore(self):
 
         assert not self.environ.is_loaded
 
+    @fixtures.switch_to_user_runtime
+    def test_temp_environment_context_manager(self):
+        with env.temp_environment(variables=[('_var0', 'val2'),
+                                             ('_var3', 'val3')],
+                                  modules=['testmod_foo']):
```

Comment:
For brevity, don't use kwargs here. You will just need to change the order of the arguments.